### DISCUSSION TITLE
Looking for dEgdT Value for Copper Indium Gallium diSelenide (CIGS) in pvlib calcparams_desoto Function

### DISCUSSION THREAD

**Discussion by OdzakSenad** [pvlib/pvlib-python#2282] [General]
Hello, everyone. I'm looking to use pvlib.pvsystem.calcparams_desoto for Copper Indium Gallium diSelenide (CIGS) modules. However, I noticed that the function's documentation leaves the value for dEgdT as unknown (dEgdT = ????). I understand that dEgdT varies depending on the indium and galium concentration in the CIGS material. Does anyone have insights on a typical dEgdT value that I could use with this function or any resources to help determine this parameter?

Thank you in advance for any help!

**Comment by cwhanse** [comment]
CIGS has a tunable bandgap from about 1.0 to 1.7 eV by varying the gallium. I would guess that the temperature dependence of the bandgap is also variable. This [reference ](https://www.researchgate.net/publication/225474301_Structural_and_optical_properties_of_thin_films_of_CuInGaSe2_semiconductor_compounds)measured dEgdT of about -7e-5 for a CIGS cell with bandgap of 1.21eV at 300K.

**Reply by OdzakSenad** [reply]
Dear cwhanse,

Thank you so much for the quick response. I really appreciate the insights, it's very helpful for me.

Best regards,

Senad
